client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -76,11 +76,6 @@ float intensity(vec3 color)
 	return (color.r + color.g + color.b) / 3.0;
 }
 
-float get_rgb_height(vec2 uv)
-{
-	return intensity(texture2D(baseTexture, uv).rgb);
-}
```

Comment:
please also remove the now unused `intensity` function here and in the object shader code